mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -16,6 +16,11 @@
                                 <FilterMenu t-if="showSearchBar" />
                             </div>
                             <div class="oe_mis_builder_action_buttons">
+                                <DatePicker
+                                    date="state.date"
+                                    onDateTimeChanged="onDateTimeChanged.bind(this)"
+                                    t-if="comparison_mode"
+                                />
```

Comment:
Move this to the previous div, since it is a filter.

Reply:
I moved to the bar, I don't know if you where looking for that exact position :thinking:

Reply:
Ah no, sorry, I meant in the same dick as the Filter menu. It's better for extensibility.